app/controllers/uploads_controller.rb

```diff
@@ -97,46 +103,83 @@ def klass
   end
 
   def requirements_messages
-    # this is a call to custom validators that are not listed inside the class
-    custom_batch_validator_messages = "#{klass.name}Validator::REQUIREMENT_DESCRIPTIONS".safe_constantize
     [Upload.valid_col_seps]
-      .push(*validation_messages)
-      .push(*custom_batch_validator_messages)
-      .compact
+      .push(*validation_messages_presence)
+      .push(*validation_messages_numericality)
+      .push(*validation_messages_uniqueness)
+  end
+
+  def batch
+    custom_batch_validator_messages = {
+      message: 'Requirement Description:',
+      value: "#{klass.name}Validator::REQUIREMENT_DESCRIPTIONS".safe_constantize
+    }
+    if custom_batch_validator_messages[:value].nil?
+      {}
+    else
+      [custom_batch_validator_messages]
+    end
```

Comment:
This if-else block doesn't seem to do anything. Am I missing something?

Reply:
That should of been removed, cleaned it up